examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -804,6 +816,37 @@ async function example_API_RefreshAuthToken(authClient: AuthClient) {
   }
 }
 
+async function example_API_TopicPublish(topicClient: TopicClient) {
+  const result = await topicClient.publish('test-cache', 'test-topic', 'test-topic-value');
+  if (result instanceof TopicPublish.Success) {
+    console.log("Value published to topic 'test-topic'");
+  } else if (result instanceof TopicPublish.Error) {
+    throw new Error(
+      `An error occurred while attempting to publish to the topic 'test-topic' in cache 'test-cache': ${result.errorCode()}: ${result.toString()}`
+    );
+  }
+}
+
+async function example_API_TopicSubscribe(topicClient: TopicClient) {
+  const result = await topicClient.subscribe('test-cache', 'test-topic', {
+    onError: () => {
+      return;
+    },
+    onItem: () => {
+      return;
```

Comment:
it would be good to have a log statement here, and do a single `publish` before you unsubscribe, so that this example will show a message being received.